optuna/storages/_in_memory.py

```diff
@@ -315,14 +315,6 @@ def get_best_trial(self, study_id: int) -> FrozenTrial:
                 )
             return self.get_trial(best_trial_id)
 
-    def get_trial_param(self, trial_id: int, param_name: str) -> float:
-
-        with self._lock:
-            trial = self._get_trial(trial_id)
-
-            distribution = trial.distributions[param_name]
-            return distribution.to_internal_repr(trial.params[param_name])
```

Comment:
These logics is in the lock. Do you intend to change the behavior of get_trial_param of InMemoryStorage in this PR?

Reply:
Sorry, I have put them back